goaxios_struct.go

```diff
@@ -81,3 +81,10 @@ type FormFile struct {
 	// Handle is an io.ReadCloser to use instead of a file path. This has an xor relationship with Path. GoAxios will close this handle after reading it. If you handle does not implement io.Closer, then you can use io.NopCloser to wrap it.
 	Handle io.ReadCloser
 }
+
+type Response struct {
+	response *http.Response
```

Comment:
All fields of this struct are meant to be exported else consumers of the package won't be able to access them. You're able to access them because you're within the same `package goaxios`

if you create another folder and call it another package say `package main` and try to use to run a request with goaxios, you won't be able to access these fields there.

Reply:
True, fixed it